mava/components/tf/networks/convolution.py

```diff
@@ -0,0 +1,29 @@
+from typing import List
+
+import sonnet as snt
+import tensorflow as tf
+
+
+class Conv1DNetwork(snt.Module):
```

Comment:
Should we put this in Mava? Not sure how generally useful it is for various projects? Definitely open to the idea, just not sure.

Reply:
I was also wondering about this. I am open to just use it locally in my project. So maybe I should move it? @KaleabTessera what do you think?

Reply:
I think we should have it in Mava, seems like a useful component that could be useful for a few systems. @DriesSmit @arnupretorius